### PR TITLE
Refine the readme

### LINKED ISSUE #151: Update readme

### PR DESCRIPTION
**Pull request by jdneo**
fix #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,81 @@
...
-## Supported Environments
+- Terraform commands: init, plan, apply, validate, refresh and destroy.
+- Visualize the terraform module.
 
-- [Microsoft Azure](https://azure.microsoft.com)
+## Commands
+
+Open the Command Palette (`Command`+`Shift`+`P` on macOS and `Ctrl`+`Shift`+`P` on Windows/Linux) and type in one of the following commands:
+
+<table>
```
Comment (testforstephen) [inline README.md]: markdown provides special syntax support for `table`. No need to write raw HTML here.

File: README.md [hunk 2]
```diff
@@ -1,76 +1,81 @@
 # Azure Terraform
+[![Build Status](https://travis-ci.com/Azure/vscode-azureterraform.svg?branch=master)](https://travis-ci.com/Azure/vscode-azureterraform)
+[![Release Status](https://vsmarketplacebadge.apphb.com/version-short/ms-azuretools.vscode-azureterraform.svg)](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureterraform)
 
 The VSCode Azure Terraform extension is designed to increase developer productivity authoring, testing and using Terraform with Azure. The extension provides terraform command support, resource graph visualization and CloudShell integration inside VSCode.
 
 ![overview](images/overview.png)
```
Comment (testforstephen) [inline README.md]: vscode will not render PNG/GIF with the relative path in MD. Please use the absolute path for these kinds of resources. Of course, you can wait and change it after the repo becomes public. The absolute path is like `https://raw.githubusercontent.com/Azure/vscode-azureterraform/master/image/overview.png`

File: README.md [hunk 3]
```diff
@@ -1,76 +1,81 @@
...
-This command will sync workspace files that meet the filter `azureTerraform.files` setting in your configuration to Azure clouddrive.
+![overview](https://raw.githubusercontent.com/Azure/vscode-azureterraform/master/images/overview.png)
 
 ## Requirements
 
 This extension requires:
 
 - [Terraform](https://www.terraform.io/downloads.html)
-- [Node.js 6.0+](https://nodejs.org) if you are using the CloudShell.
+- [Node.js 6.0+](https://nodejs.org) if you are using the Cloud Shell.
```
Comment (jeffreyCline) [inline README.md]: if you are using Cloud Shell.

File: README.md [hunk 4]
```diff
@@ -1,76 +1,81 @@
...
-
-- Windows support for dot has some unhandled exception cases.  We are working to improve this area.
-- We do not support private registry. The test container has to be on docker hub at this time.
-
 ## Release Notes
 
 Refer to [CHANGELOG](CHANGELOG.md)
+
+## Telemetry
+VS Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=528096&clcid=0x409) to learn more. If you don’t wish to send usage data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false`. Learn more in our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
```
Comment (jeffreyCline) [inline README.md]: If you would like to opt out of sending telemetry data to Microsoft, update the `telemetry.enableTelemetry` setting to `false` in the **File** -> **Preferences** -> **Settings**. To learn more see our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).

Comment (jdneo) [reply README.md]: Thanks, updated.

**Pull request merged**